src/plugins/foxPage/hooks/useOtherOpportunities.ts

```diff
@@ -46,16 +42,6 @@ export const useOtherOpportunities = (assetId: AssetId) => {
     selectHighestBalanceAccountIdByLpId(state, highestBalanceLpAccountIdFilter),
   )
 
-  const farmingv5EarnOpportunityFilter = useMemo(
```

Comment:
No idea why we were selecting the *earn* opportunity (metadata + user data parsed as `EarnOpportunityType`) when all we needed was metadata